versions/linea-mainnet.env

```diff
@@ -1,4 +1,4 @@
-LINEA_BESU_TAR_GZ=25.2-delivery50
+LINEA_BESU_TAR_GZ=25.3.0-linea2
```

Comment:
```suggestion
LINEA_BESU_TAR_GZ=25.3-delivery51
```

Reply:
@macfarla suggestion is correct, Linea Mainnet is still using the Linea Besu fork